testpack/moduleA/build.gradle

```diff
@@ -1,37 +1,11 @@
 // Copyright 2021 The Terasology Foundation
 // SPDX-License-Identifier: Apache-2.0
-import org.reflections.Reflections
-import org.reflections.scanners.ResourcesScanner
-import org.reflections.scanners.SubTypesScanner
-import org.reflections.scanners.TypeAnnotationsScanner
-import org.reflections.serializers.JsonSerializer
-
 apply plugin: 'java'
 
-dependencies {
-    implementation project(":testpack:testpack-api")
-}
-
-def reflect() {
-    def resourcesPath = sourceSets.find { it.name == 'main' }.resources.srcDirs
-    def classPath = files { compileJava.destinationDir }
-    URL[] scanUrls = classPath.collect { it.toURI().toURL() } + resourcesPath.collect {
-        it.toURI().toURL()
-    }
-    URL[] classLoaderUrls = scanUrls + compileJava.classpath.collect { it.toURI().toURL() }
-
-    ClassLoader classLoader = new URLClassLoader(classLoaderUrls, getClass().getClassLoader())
-    org.reflections.Configuration config = new org.reflections.util.ConfigurationBuilder()
-    config.addClassLoader(classLoader)
-    config.setUrls(scanUrls)
-
-    config.addScanners(new ResourcesScanner(), new SubTypesScanner(false), new TypeAnnotationsScanner().filterResultsBy())
-    Reflections reflections = new Reflections(config)
-    JsonSerializer serializer = new JsonSerializer()
-    def outPath = compileJava.destinationDir.toString() + "/manifest.json"
-    serializer.save(reflections, outPath)
+jar {
+    version = ''
 }
 
-compileJava.doLast {
-    reflect()
-}
```

Comment:
These lines should not have been removed: they are a part of how gestalt v7 works. Gestalt v8 uses annotation processors instead, which is why the code was removed on `develop`.

Gestalt 7 caches a list of all potential classes of interest using the `org.reflections` library. This cached list was intended to significantly reduce the costly impact of runtime reflection, as we would already know which classes to look for. On Android (which Terasology doesn't support but [Destination Sol](https://github.com/MovingBlocks/DestinationSol) does), this was essential since the reflections library wasn't supported on Android at the time.

Reply:
Apologies for that, the previous commit caused the code in this case to fail, but it didn't click with me at the time that it was for that reason the original code here was not working.